config/settings.yml

```diff
@@ -2,10 +2,19 @@
 :git_mirror:
   :branch_mirror_defaults: {}
   :branch_mirror_overrides:
-    # The following repos don't have release branches, so don't mirror master to the ##.y branch
+    # container repos don't follow branch naming conventions, so don't mirror master to a release branch
+    # and also include any container-specific branches directly
+    container-httpd:
+      master:
+    container-memcached:
+      master:
+    container-postgresql:
+      master:
+      10-el8:
```

Comment:
In summary, when I added this, since it's not in `branch_mirror_defaults` it was ignored, which is what this PR fixes.